src/pages/2021/sponsorlogistics.mdx

```diff
@@ -2,9 +2,105 @@ import { SiteLayout } from "../../components/layout/SiteLayout";
 
 <SiteLayout title="Schedule">
 
-# Conference Sponsor Logistics
+# 2021 Conference Sponsor Logistics
 
-The oSTEM conference team is hard at work in the planning stages of the 2021
-conference. Check back here later for the full list of sponsors.
+Thank you for sponsoring the oSTEM 2021 Virtual Annual Conference! This page contains the latest information regarding conference planning and sponsor participation. Please be sure to revisit this page to receive the the most up-to-date information. 
+
+# Table of Contents
+
+- [Key Deadlines and Milestones](#deadlines)
+
+- [Conference Schedule and Programming Highlights](#schedule)
+
+- [Sponsored Workshops](#workshops)
+
+    - [50-minute Sponsored Workshop Submission](#submit)
+
+    - [What should you submit?](#submit-ideas)
+
+    - [Are you stuck and don't know what to present?](#present-ideas)
+
+- [Organization Logo Requirements](#)
+
+- [Program Advertisement Requirements](#)
+
+- [Virtual Platforms](#)
+
+- [Career Fair via Brazen](#)
+
+- [Contact Information, Resumes/CVs](#)
+
+- [oSTEM Career Center](#)
+
+- [Awards Information](#)
+
+
+# <a name="deadlines"></a> Key Deadlines and Milestones
+
+## Sponsor Key Deadlines and Milestones
+
+# <a name="schedule"></a> Conference Schedule and Programming Highlights
```

Comment:
Hmmm, this feels a bit weird. The h1 -> h2 -> h1 just looks jumbled. Do we not have this information yet? Or did it maybe not make it's way onto this page from the Notion doc?

<img width="1616" alt="Screen Shot 2021-08-15 at 9 42 21 PM" src="https://user-images.githubusercontent.com/773453/129512328-da5a72d3-2ba3-467e-9928-3dde26d7dcf1.png">

Reply:
I will ask Emily tonight at the core team meeting, expect an answer after 5:30 pm PST

Reply:
Emily wants a table similar to the one here: https://ostem.org/page/conference-2020-conference-sponsor-logistics/

Reply:
Hi all - I don't check github as often as I check slack. If you need feedback from me (sponsorship chair), please slack me directly.